modules/cluster-logging-updating-logging-to-current.adoc

```diff
@@ -0,0 +1,220 @@
+
+.Procedure
+
+. Update the OpenShift Elasticsearch Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-Operators-redhat` project.
+
+.. Click the *OpenShift Elasticsearch Operator*.
+
+.. Click *Subscription* -> *Channel*.
+
+.. In the *Change Subscription Update Channel* window, select *stable-5.x* and click *Save*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+.. Verify that the OpenShift Elasticsearch Operator version is 5.x.x.
++
+.. Wait for the *Status* field to report *Succeeded*.
+
+. Update the Red Hat OpenShift Logging Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-logging` project.
+
+.. Click the *Red Hat OpenShift Logging Operator*.
+
+.. Click *Subscription* -> *Channel*.
+
+.. In the *Change Subscription Update Channel* window, select *stable-5.x* and click *Save*.
+
+.. Wait for a few seconds, then click *Operators* -> *Installed Operators*.
++
+.. Verify that the Red Hat OpenShift Logging Operator version is 5.y.z
++
+.. Wait for the *Status* field to report *Succeeded*.
+
+. Check the logging components:
+
+.. Ensure that all Elasticsearch pods are in the *Ready* status:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-logging --selector component=elasticsearch
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                            READY   STATUS    RESTARTS   AGE
+elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk   2/2     Running   0          31m
+elasticsearch-cdm-1pbrl44l-2-5c6d87589f-gx5hk   2/2     Running   0          30m
+elasticsearch-cdm-1pbrl44l-3-88df5d47-m45jc     2/2     Running   0          29m
+----
++
+.. Ensure that the Elasticsearch cluster is healthy:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch elasticsearch-cdm-1pbrl44l-1-55b7546f4c-mshhk -- health
+----
++
+[source,json]
+----
+{
+  "cluster_name" : "elasticsearch",
+  "status" : "green",
+}
+----
+
+.. Ensure that the Elasticsearch cron jobs are created:
++
+[source,terminal]
+----
+$ oc project openshift-logging
+----
++
+[source,terminal]
+----
+$ oc get cronjob
+----
++
+[source,terminal]
+----
+NAME                     SCHEDULE       SUSPEND   ACTIVE   LAST SCHEDULE   AGE
+elasticsearch-im-app     */15 * * * *   False     0        <none>          56s
+elasticsearch-im-audit   */15 * * * *   False     0        <none>          56s
+elasticsearch-im-infra   */15 * * * *   False     0        <none>          56s
+----
+
+.. Verify that the log store is updated to 5.x and the indices are `green`:
++
+[source,terminal]
+----
+$ oc exec -c elasticsearch <any_es_pod_in_the_cluster> -- indices
+----
++
+.. Verify that the output includes the `app-00000x`, `infra-00000x`, `audit-00000x`, `.security` indices.
```

Comment:
By default, the audit logs are not enabled.  audit-00000x appear only the audit input are defined in ClusterLogForwarder

```
apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: instance
  namespace: openshift-logging
spec:
  pipelines:
    - name: all-to-defaultES
      inputRefs:
      - infrastructure
      - application
      - audit
      outputRefs:
      - default

```